csvtail.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+import sys
+
+
+DESCRIPTION = 'csvtail - prints header and last lines of input.'
+EXAMPLES = 'example: cat file.csv | csvtail -n -100 skip first 100 rows and print file.csv till the end.'
+
+def main(args):
+    input_stream = open(args.file, 'r') if args.file else sys.stdin
+    output_stream = open(args.output_file, 'w') if args.output_file else sys.stdout
+
+    output_stream.write(input_stream.readline())
+    if args.rows_count < 0:
+	for _ in range(-args.rows_count):
+            input_stream.readline()
+        for row in input_stream:
+            output_stream.write(row)
+    else:
+        lines = input_stream.readlines()
```

Comment:
This line loads the whole file into memory. Imagine, what would happen if the file size is several GB.